serve_test.go

```diff
@@ -514,7 +514,7 @@ func TestServer_RunTLS(t *testing.T) {
 			}()
 
 			// wait for the server to start
-			conn, err := net.DialTimeout("tcp", s.Server.Addr, 2*time.Second)
+			conn, err := net.DialTimeout("tcp", s.Server.Addr, 5*time.Second)
 			if err != nil {
```

Comment:
Yeah why not

Reply:
I admittedly did not look through the entirety of of `Dial` to see if refused meant timeout. But 🤷. Feels like the server isn't starting up quick enough to me personally.